demos/data-action/action-setup.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
In https://github.com/atk4/ui/commit/ac4b2512bed644bd92f47cedb0886ba8bd127ccd I have moved this file to `jsactions2.php` to not need the phpunit exclude. I prefer it also when manually testing some demos from the folders, I do not have to think if this file is functional on it's own or not.

Reply:
my bad. I screw up during resolve conflict. Thanks.